compiler/optimizations/gpuTransforms.cpp

```diff
@@ -760,6 +792,37 @@ bool GpuizableLoop::callsInBodyAreGpuizableHelp(BlockStmt* blk,
 
       FnSymbol *fn = call->resolvedFunction();
 
+      // nonLocalAccess function is really complicated, on a quick look it has:
+      // - allocation (RAD cache)
+      // - atomics
+      // - communication
+      // I tried adding stubs for these to just to get it to compile, but it
+      // spiraled out too fast. So, we'll just make a new copy for GPU here that
+      // just errors. We don't expect this function to be called until we have
+      // GPU-driven communication.
+      if (fn->name == astr("nonLocalAccess")) {
```

Comment:
Not sure if we care but could this potentially get triggered if a user wrote their own function named `nonLocalAccess`?

Would it work if we just checked that `fn` was in standard module rather than user code.

Reply:
I am adding a pragma for this and not doing name-based matching anymore.